Diagnostic/diagnostic.py

```diff
@@ -1166,6 +1175,51 @@ def get_deployment_id():
 
     return identity
 
+class Watcher():
```

Comment:
I suspect we'll want to add more things-to-be-watched over time, which means this will grow. Might make sense to start off right with Watcher and the fstab_handler being in a separate module from the rest of LAD; might make unit testing easier as well.